projects/storefrontlib/src/lib/store-finder/components/store-finder-grid/store-finder-grid.component.spec.ts

```diff
@@ -136,19 +132,14 @@ describe('StoreFinderGridComponent', () => {
 
   function configureTestBed(): void {
     const bed = TestBed.configureTestingModule({
```

Comment:
Can we make this test consistent with others? This `function configureTestBed()` and `createComponent()` can be replaced by `beforeEach`. Please check how other tests are implemented.

Reply:
I refactored this unit test.